appveyor.yml

```diff
@@ -0,0 +1,68 @@
+version: 3.0.5.{build}
```

Comment:
We can use a notation like `version: {branch}-{build}`, that way we wont have to change it for releases.